shuttle/templates/request_shuttle/request_shuttle.html

```diff
@@ -12,16 +12,38 @@ <h3>Your Position In The Waitlist: <span class="badge badge-pill badge-warning w
                 <div class="card-body">
                     <h4>Pick Up: <span class="badge badge-pill badge-warning pick-up-status">{{ active_requests['requested-pick-up'] }}</span></h4>
                     <h4>Drop Off: <span class="badge badge-pill badge-warning drop-off-status">{{ active_requests['requested-drop-off'] }}</span></h4>
-                    <input class="btn btn-danger btn-sm delete-btn" type="submit" value="Delete Request">
+                    <input class="btn btn-danger btn-sm delete-request" type="submit" value="Delete Request">
                 </div>
             </div>
         </div>
     {% elif shuttle_requestable == True %}
         <div class="center-page shuttle-margins">
             <h3>Number Of Users Waiting For On Call Shuttle: <span class="badge badge-pill badge-warning waitlist">{{ active_requests['waitlist-num'] }}</span></h3>
         </div>
+        <div class="center-page">
+            {% if phone_number %}
+                <div class="align-side-by-side">
+                    <h4>Phone number On Record: <b>{{ phone_number }}</b></h4>
+                </div>
+                <div class="align-side-by-side">
+                    <input class="btn btn-danger btn-sm delete-number" type="button" value="Delete">
+                </div>
+            {% else %}
+                <form>
+                    <div class="align-side-by-side">
+                        <h4>Phone number:</h4>
```

Comment:
You could combine these two <h4>s into just one, don't need to specify if the phone is "on record" or not since it will just auto-populate if it is